Sources/AblyChat/RoomLifecycleManager.swift

```diff
@@ -150,6 +339,26 @@ internal actor RoomLifecycleManager<Channel: RoomLifecycleContributorChannel> {
 
         // CHA-RL1g1
         changeStatus(to: .attached)
```

Comment:
Not sure why is hardcoded `.attached` here (I understand this comes not from this PR, but still curious if this is something that could be addressed here).

Reply:
Sorry, I messud up it with being inside `private func changeStatus(to new: RoomLifecycle, error: ARTErrorInfo? = nil) {`